.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Bug Report - Netdata Cloud
+    url: https://github.com/netdata/netdata-cloud/issues/new/choose
+    about: Create a report to help us improve our SaaS offering
+  - name: Help
+    url: https://community.netdata.cloud/c/support/13
+    about: We offer community-driven support on our Community forums
+  - name: Feature Request
+    url: https://community.netdata.cloud/c/feature-requests/7
```

Comment:
So it is one category for both agent and cloud feature requests? Is this on purpose like it is one project netdata?

I opened the page and it is not clear what _requests_ ar ecloud related/agent related?

<img width="1055" alt="Screenshot 2020-11-19 at 14 32 11" src="https://user-images.githubusercontent.com/22274335/99660860-0433ab00-2a74-11eb-8ce0-242252b51bca.png">

---

I think we are gonna _group_ them using labels. Discourse has tags (~ github labels). That is the idea?

Reply:
Hm, I didn't thought about using categories for feature requests. Do you think it's better to group them by product, as we do in the support category @ilyam8 ?

Reply:
For example, in Agent development category, I didn't implement sub-categories for collectors, exporters, etc. as I had initially planned, it's better to simply use tags. Each sub-category ads an additional step in posting a topic (because I have to navigate to the correct sub) .

Reply:
My first instinct on this is to have a generic feature-request label on Discourse, and then sub-labels for cloud and agent. People who know where the FR needs to go can put it there, those who don’t can just open it under the generic label and we can move it to the right place.

Reply:
Labels (tags) will do for me (if UI allows to filter based on them).

Reply:
I created the sub-categories. The user (or us) can even segment on the component that the feature request refers to via using the same tags we use in other categories (e.g support).

Reply:
Here is a list of tags: https://community.netdata.cloud/tags

Reply:
I think it good and it can be revised if needed (as I understand it's flexible enough). 

Agent feature requests can be very specific but we need to examine each one taking into account the "cloud big picture". Similarly a cloud feature request can internally generate/involve a  "low-level agent feature"

Reply:
Cool, let's have in mind that this is a community feature, meant to give the community a voice. Internal project management should be absolutely decoupled.
I envision that given the feature requests category, the product team will take into account what the community wants and revise the roadmap. But the tracking of the roadmap tasks is another process/system.

Reply:
Based on personal experience, the right approach here for the segmentation is to use the forum to figure out what the user-facing feature should work like, and then open GitHub issues for the individual parts that need to be done and have them block each other as appropriate.

That way users get to contribute to the design of the actual parts they will see, but don’t have to care about the internal project management aspects.

Reply:
And, of course, once we have the issues on GitHub, back-link them to the forum so that users on the forum can find them easily.

Reply:
I think we are in alignement. I have created the appropriate categories on the forums.